doc/getting-started/building-from-source.rst

```diff
@@ -79,8 +79,16 @@ Install the necessary tools for the following systems:
           libblkid-dev \
           e2fslibs-dev \
           pkg-config \
-          libnuma-dev
+          libnuma-dev \
+          liblz4-tool \
+          flex \
+          bison
+
      $ sudo pip3 install kconfiglib
+     $ wget https://acpica.org/sites/acpica/files/acpica-unix-20191018.tar.gz
```

Comment:
Instead of linking to the specific (and potentially outdated) tar file, we could link to their download page and tell folks to download the "Unix Format Source Code and Build Environment with Intel license" (https://acpica.org/downloads)

Reply:
It has been fully tested by using the specific tar file, it will have a task to manage the version later.

Reply:
Can you separate the installation iof the `iasl` tool from the `kconfiglib` above. I want it to be clear that these are different tools.

I also think it would be worthwhile mentionning here that the tool serves two purposes, one is as a direct runtime dependency of `acrn-dm` and it's also used during the build process to pre-compile ACPI tables.

I would also make it explicit to **not** use the distro package unless they verify it is an appropriate version.